shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -433,7 +433,7 @@ public void itForwardsOnRequestPermissionsResultToFlutterEngine() {
   public void
       itSendsInitialRouteFromIntentOnStartIfNoInitialRouteFromActivityAndShouldHandleDeeplinking() {
     Intent intent = FlutterActivity.createDefaultIntent(RuntimeEnvironment.application);
-    intent.setData(Uri.parse("http://myApp/custom/route?query=test"));
+    intent.setData(Uri.parse("http://myApp/custom/route?query=test#fragment"));
```

Comment:
Can you create a new test instead of modifying existing test? we had a regression before that the new change causes old test the fail, but we didn't catch that because the old test is updated to use the new input

Reply:
Make sense, will add.
Hope I can up the changes in the next hour.